maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gittest/src/main/java/org/apache/maven/scm/provider/git/GitScmTestUtils.java

```diff
@@ -88,6 +92,28 @@ public static void deleteAllDirectories(File startDirectory, String pattern) thr
         }
     }
 
+    /**
+     * Creates a new file below a new temporary directory and copies the content of a classpath resource into it.
+     * The caller is responsible for deleting the temporary directory afterwards.
+     * @param clazz
```

Comment:
either describe these arguments or don't include a javadoc tag

Reply:
Let's say we disagree here, however I added descriptions to the arguments in this case.